### PR TITLE
fixing the names SimpleRNNLearner and AutoencoderLearner

### PR DESCRIPTION
**Pull request by josd**
Fixing the names in the notebook notebooks/chapter19.RNN.ipynb according to the names used in deep_learning4e.py

As an aside, the training run with SimpleRNNLearner gives a lower validation accuracy:
```
...
Epoch 10/10
 - 64s - loss: 0.5762 - accuracy: 0.6814 - val_loss: 0.6450 - val_accuracy: 0.5956
```

It was tested with the following packages:
```
Name: tensorflow
Version: 1.14.0
Summary: TensorFlow is an open source machine learning framework for everyone.
Home-page: https://www.tensorflow.org/
Author: Google Inc.
Author-email: packages@tensorflow.org
License: Apache 2.0
Location: /usr/local/lib/python3.6/dist-packages
Requires: grpcio, termcolor, gast, wheel, tensorboard, six, keras-preprocessing, tensorflow-estimator, wrapt, protobuf, absl-py, astor, keras-applications, numpy, google-pasta
Required-by: tensorflow-serving-api, Entailment
---
Name: Keras
Version: 2.3.0
Summary: Deep Learning for humans
Home-page: https://github.com/keras-team/keras
Author: Francois Chollet
Author-email: francois.chollet@gmail.com
License: MIT
Location: /usr/local/lib/python3.6/dist-packages
Requires: six, scipy, h5py, keras-preprocessing, keras-applications, pyyaml, numpy
Required-by: 
```

### CODE REVIEW

**Comment by antmarakis** [comment]
Thanks for the PR, much appreciated! Could you remove the warning messages? I think these lines should do it:

```
import warnings
warnings.filterwarnings("ignore", category=FutureWarning)
```

**Comment by josd** [comment]
Thanks Antonis and there is a new commit for that.
Meanwhile I updated keras, scipy and numpy and the eval accuracy is now indeed around 80%

```
$ pip show keras scipy numpy
Name: Keras
Version: 2.3.1
Summary: Deep Learning for humans
Home-page: https://github.com/keras-team/keras
Author: Francois Chollet
Author-email: francois.chollet@gmail.com
License: MIT
Location: /usr/local/lib/python3.6/dist-packages
Requires: pyyaml, keras-preprocessing, keras-applications, scipy, h5py, numpy, six
Required-by: hyperas
---
Name: scipy
Version: 1.3.1
Summary: SciPy: Scientific Library for Python
Home-page: https://www.scipy.org
Author: None
Author-email: None
License: BSD
Location: /usr/local/lib/python3.6/dist-packages
Requires: numpy
Required-by: tensor2tensor, scikit-learn, projectq, Keras, jaxlib, hyperopt, gym
---
Name: numpy
Version: 1.17.3
Summary: NumPy is the fundamental package for array computing with Python.
Home-page: https://www.numpy.org
Author: Travis E. Oliphant et al.
Author-email: None
License: BSD
Location: /usr/local/lib/python3.6/dist-packages
Requires: 
Required-by: torchvision, torch, tfds-nightly, tf-agents, tensorflow, tensorflow-probability, tensorflow-hub, tensorflow-datasets, tensorboard, tensor2tensor, scipy, scikit-learn, projectq, pandas, opt-einsum, opencv-python, matplotlib, kfac, Keras, Keras-Preprocessing, Keras-Applications, jaxlib, jax, hyperopt, h5py, gym, autograd, Entailment
```
